app/client/src/components/shared/MapMouseEvents/index.js

```diff
@@ -53,7 +53,16 @@ function MapMouseEvents({ map, view }: Props) {
         .then((res) => {
           // get and update the selected graphic
           const graphic = getGraphicFromResponse(res);
+
           if (graphic && graphic.attributes) {
+            // if upstream watershed is clicked:
+            // set the view highlight options to 0 fill opacity
+            if (graphic.layer.id === 'upstreamWatershed') {
+              view.highlightOptions.fillOpacity = 0;
+            } else {
+              view.highlightOptions.fillOpacity = 1;
+            }
```

Comment:
I noticed that when you click on the upstream watershed with an area waterbody below it, that the area waterbody only has the outline highlighted. 

If we move this block of code down to the `view.popup.watch('selectedFeature'` section, then the upstream watershed graphic will only have the outline highlighted and the area waterbody graphic will have the entire graphic highlighted. 

The final code would look something like:
```
view.popup.watch('selectedFeature', (graphic) => {
      // check if monitoring station is clicked, load the popup and call the waterqualitydata service
      if (
        graphic &&
        graphic.layer &&
        graphic.layer.id === 'monitoringStationsLayer' &&
        graphic.attributes &&
        graphic.attributes.fullPopup === false
      ) {
        loadMonitoringLocation(graphic, services);
      }

      // if upstream watershed is clicked:
      // set the view highlight options to 0 fill opacity
      if (graphic?.layer?.id === 'upstreamWatershed') {
        view.highlightOptions.fillOpacity = 0;
      } else {
        view.highlightOptions.fillOpacity = 1;
      }
    });
```

Reply:
@cschwinderg Good call, I overlooked the 'selectedFeature' event. By using that we avoid most of the issues with this method of modifying the highlight opacity.  Thanks!